### PR TITLE
Removes notes about Bower & fix canary instructions

### LINKED ISSUE #68: Installing Canary build docs
https://cli.emberjs.com/release/appendix/common-issues/

Installing Canary Build docs reference bower install, which I think is outdated info?

### PR DESCRIPTION
**Pull request by villander**
This fixes #68 

cc: @jenweber

### CODE REVIEW

File: guides/appendix/common-issues.md [hunk 1]
```diff
@@ -34,7 +27,7 @@ git config --global url."https://".insteadOf git://
 
 In most cases you should use a stable release, but if you need to install a canary version to test beta features, you'd do it like this:
 
-For Ember: `bower install ember#canary --resolution canary`
+For Ember: `npm install --save-dev https://s3.amazonaws.com/builds.emberjs.com/canary/shas/106bb03e87f982657edd4595652579bfeae3f06a.tgz`
```
Comment (rwjblue) [inline guides/appendix/common-issues.md]: I don't think we should list a specific canary SHA here like this. Instead, can you update the prose to something like:

```md
In most cases you should use a stable release, but if you need to 
install a canary version to test some new feature you can update
the `ember-source` entry in your `package.json` to use the latest
canary tarball URL. You can find the most recent canary tarball URL
by way of the 
[ember-source-channel-url](https://github.com/ember-cli/ember-source-channel-url)
package.

That might look like:

`npx ember-source-channel-url canary --write`
```

Comment (villander) [reply guides/appendix/common-issues.md]: fixed @rwjblue

**Comment by rwjblue** [comment]
@villander - I think you need to add `ember-source-channel-url` to the allowed dictionary, it is currently failing the build (you can run via `npm run lint:md` I think).

**Comment by villander** [comment]
fixed @rwjblue, we can merge 🎉

**Pull request merged**